keyboards/qwertyydox/keymaps/default/rules.mk

```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
```

Comment:
This block isn't needed, and is a holdover from the old makefile system. 

Please remove it